etstool.py

```diff
@@ -94,7 +94,9 @@
     "traitsui",
     "configobj",
     "coverage",
-    "pytables",
+    # This version is pinned due to the latest build causing failures on the
+    # travis CI see issue traitsui/#1156.
```

Comment:
The comment makes it look like the issue was caused by traitsui, but it isn't. Could we remove the reference please? I think it is okay to reference buildsystem if you must reference something.